src/js/components/Select/doc.js

```diff
@@ -132,6 +132,10 @@ export const doc = Select => {
       PropTypes.oneOf(['small', 'medium', 'large', 'xlarge']),
       PropTypes.string,
     ]).description('The size of the select.'),
+    height: PropTypes.oneOfType([
```

Comment:
I think `dropHeight` might be a better term. It would align with other places like TextInput `dropAlign`.